repo-projects/source-build-reference-packages.proj

```diff
@@ -17,4 +14,16 @@
     <BuiltSdkPackage Include="Microsoft.Build.Traversal" />
   </ItemGroup>
 
+  <!-- Copy reference only packages to prereqs/package/reference -->
+  <Target Name="CopyReferenceOnlyPackages"
+          AfterTargets="RepoBuild"
+          BeforeTargets="CleanupRepo" >
+    <ItemGroup>
+      <_ReferenceOnlyPackages Include="$(RepoArtifactsDir)packages\$(Configuration)\ReferenceOnly\**\*" />
```

Comment:
nit: why the `**\*` glob? I would expect the ReferenceOnly dir to be a flat folder.